index.html

```diff
@@ -176,7 +176,7 @@ <h4 class="about-me-title">About me</h4>
 
                 <div class="card framework-card">
                     <div class="framework card-img">
-                        <img src="pics/framework-icon.png" alt="icon">
+                        <img src="pics/framework-icon.png" alt="icon framework">
                     </div>
                     <div class="card-details">
```

Comment:
**Resize Text**:
* Please ensure that Text, images, and other content do not overlap.
Kindly fix that.

![image](https://user-images.githubusercontent.com/10332499/156136049-f6f2bcf0-c0b1-4643-8c88-d35360ca9dcf.png)